CoreFeatures/Database-postgresql/PostgresSchema/src/main/java/info/smart_tools/smartactors/database_postgresql/postgres_schema/search/PostgresQueryWriterResolver.java

```diff
@@ -39,15 +39,15 @@ public void addQueryWriter(final String name, final QueryWriter writer) {
         queryWriters.put(name, writer);
     }
 
-    /**
-     * Resolves valid field path in dotted notation ('a.b') to jsonb notation ('document#>{a,b}').
-     * @param name name of field path in dotted notation from the criteria.
-     * @return a FieldPath object with valid field path
-     * @throws QueryBuildException when field path is invalid
-     */
-    public FieldPath resolveFieldName(final String name) throws QueryBuildException {
-        return PostgresFieldPath.fromString(name);
-    }
+//    /**
+//     * Resolves valid field path in dotted notation ('a.b') to jsonb notation ('document#>{a,b}').
+//     * @param name name of field path in dotted notation from the criteria.
+//     * @return a FieldPath object with valid field path
+//     * @throws QueryBuildException when field path is invalid
+//     */
+//    public FieldPath resolveFieldName(final String name) throws QueryBuildException {
+//        return PostgresFieldPath.fromString(name);
+//    }
```

Comment:
As I remember it was a common policy that we should not keep code which is commented out.
Is there any reason to keep this code?

Reply:
Ok, I'll fix it